doc/source/user/c-info.python-as-glue.rst

```diff
@@ -123,7 +113,8 @@ F2py allows you to automatically construct an extension module that
 interfaces to routines in Fortran 77/90/95 code. It has the ability to
 parse Fortran 77/90/95 code and automatically generate Python
 signatures for the subroutines it encounters, or you can guide how the
-subroutine interfaces with Python by constructing an interface-definition-file (or modifying the f2py-produced one).
+subroutine interfaces with Python by constructing an interface-definition-file
+(or modifying the f2py-produced one).
```

Comment:
I'm trying hard not to rant about parenthetical phrases here, with mixed success ;)